packages/cli-platform-apple/src/commands/buildCommand/createBuild.ts

```diff
@@ -34,14 +32,7 @@ const createBuild =
       installedPods = true;
     }
 
-    // if project is freshly created, revisit Xcode project to verify Pods are installed correctly.
-    // This is needed because ctx project is created before Pods are installed, so it might have outdated information.
-    if (installedPods) {
-      const recheckXcodeProject = findXcodeProject(fs.readdirSync(sourceDir));
-      if (recheckXcodeProject) {
-        xcodeProject = recheckXcodeProject;
-      }
-    }
+    xcodeProject = getXcodeProjectAndDir(platform, installedPods).xcodeProject;
```

Comment:
In this file we could move the line 19 here:
```
let {xcodeProject, sourceDir} = getXcodeProjectAndDir(platform);
```
as both `xcodeProject` and `sourceDir` are not used until later on

Reply:
Fixed